README.md

```diff
@@ -116,90 +123,137 @@ Maven:
 ```xml
 <dependency>
     <groupId>org.globalnames</groupId>
-    <artifactId>parser</artifactId>
+    <artifactId>parser_2.11</artifactId>
```

Comment:
Actually should be 
```
<dependency>
    <groupId>org.globalnames</groupId>
    <artifactId>gnparser_2.11</artifactId>
    <version>0.1.0</version>
</dependency>
```

Also, SBT dependency should be: `libraryDependencies += "org.globalnames" % "gnparser_2.11" % "0.1.0"`

Take a look at: http://search.maven.org/#artifactdetails%7Corg.globalnames%7Cgnparser_2.11%7C0.1.0%7Cjar